experimental/terraform/README.md

```diff
@@ -1,42 +1,187 @@
-# Experimental Terraform setup
-
-This folder contains terraform scripts to provision all of the infrastructure in a Four Keys GCP project. 
-
-**DO NOT USE!** This isn't ready for production yet (though it's close!)
-
-## How to use
-1. run `setup.sh`; this will:
-  * create a project for Four Keys
-  * purge all terraform state [useful during tf development]
-  * build containers using Cloud Build
-  * create a `terraform.tfvars` file
-  * invoke terraform
-1. run the following commands to retrieve values needed for your SCM:
-  * ```
-    echo `terraform output -raw event_handler_endpoint`
-    echo `terraform output -raw event_handler_secret`
+> :information_source: This README is intended to replace `/setup/README.md`. 
+> When it's time to promote the Terraform installer to become the preferred
+> installation method (hopefully soon!): 
+> * move the current contents of `/setup` to `/setup/deprecated`
+> * move the contents of this directory to `/setup`
+> * remove this note
+
+# Installation guide
+
+This guide describes how to set up Four Keys with your GitHub or GitLab project. The main steps are:
+
+1.  [Runing the setup script](#running-the-setup-script)
+1.  Integrating with your GitHub or Git Lab repo by:
+    1.  [Collecting changes data](#collecting-changes-data)
+    1.  [Collecting deployment data](#collecting-deployment-data)
+    1.  [Collecting incident data](#collecting-incident-data)
+
+## Before you begin
+
+1.  Install [GCloud SDK](https://cloud.google.com/sdk/install).
+1.  You must be owner on a Google Cloud project that has billing enabled. You may either use this project to house the architecture for the Four Keys, or you will be given the option to create new projects. If you create new projects, the original Google Cloud project will NOT be altered during set up, but the billing information from this parent project will be applied to any projects created.
```

Comment:
Should we make the billing information bold?  Important piece of information.

Reply:
done